### PR TITLE
ISSD-1494 [Marketplace] Create Route For App Details and App Licenses Tabs

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/AppOutlet.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+			<ClayButton
+				className="mb-2"
+				displayType="unstyled"
+				onClick={() => navigate('/')}
+			>
+				<ClayIcon className="mr-2" symbol="order-arrow-left" />
+				Back to My Apps
+			</ClayButton>
+			<ClayNavigationBar className="mb-4" triggerLabel={active}>
+				<ClayNavigationBar.Item active={active === 'Details'}>
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/AppOutlet.tsx:30]: Please, use an enum to 'Details' and 'Licenses'

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added a suggestion

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/AppOutlet.tsx (hunk 1 above)
Comment (claricebrasil) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/AppOutlet.tsx]: Added, thank you!

**Comment by eliasantosf** [comment]
Just starting reviewing :)

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/PurchasedAppsDashboardTableRow.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 .dashboard-table-row {
+	cursor: pointer;
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/PurchasedAppsDashboardTableRow.scss:2]: Please, add a line break after cursor:pointer;
```suggestion
	cursor: pointer;
    
```

**Review by juliocezar96 — APPROVED** [review]
Good job! 🤙

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/PurchasedAppsDashboardTableRow.tsx [hunk 3]
```diff
@@ -136,7 +139,7 @@ export function PurchasedAppsDashboardTableRow({
 				</div>
 			</ClayTable.Cell>
 
-			<ClayTable.Cell>
+			<ClayTable.Cell onClick={(event) => event.stopPropagation()}>
 				<DropDown
 					trigger={
 						<ClayButton displayType="secondary">
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/PurchasedAppsDashboardTableRow.tsx:145]: If possible, could you to add the lib i18n for this file?

 - Manage
 - Create License Key
 - Access Console
 - Download App
 - This order must be completed before downloading this app.
 - The order must be completed before licensing this app.

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/AppOutlet.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+	return (
+		<div>
+			<ClayButton
+				className="mb-2"
+				displayType="unstyled"
+				onClick={() => navigate('/')}
+			>
+				<ClayIcon className="mr-2" symbol="order-arrow-left" />
+				Back to My Apps
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/AppOutlet.tsx:29]: Please, use i18n to this file.
 - Back to My Apps
 - Details
 - Licenses

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions

**Comment by joanagrego** [comment]
Just starting reviewing :)

**Comment by joanagrego** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
LGTM :fire

resent: https://github.com/brianchandotcom/liferay-portal/pull/143179

**Pull request closed without merging**